mulighetsrommet-api/src/main/resources/web/openapi.yaml

```diff
@@ -2444,10 +2444,6 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/SanityRegelverkLenke"
-        nokkelinfoKomponenter:
-          type: array
-          items:
-            $ref: "#/components/schemas/SanityNokkelinfo"
```

Comment:
Kan sikkert fjerne typedekklarasjonen for `SanityNokkelinfo` fra spec'en også?

Reply:
Jepp